source/user-manual/reference/daemons/wazuh-authd.rst

```diff
@@ -62,6 +72,16 @@ The ``wazuh-authd`` program can automatically add a Wazuh agent to a Wazuh manag
 +------------------+-------------+-----------------------------------------------------------------------------------------+
 | **-L**           | Force insertion even though agent limit has been reached.                                             |
 +------------------+-------------------------------------------------------------------------------------------------------+
+| **-C**           | Specify the manager certificate validity in days.                                                     |
++------------------+-------------------------------------------------------------------------------------------------------+
+| **-B**           | Set the manager certificate key size in bits.                                                         |
++------------------+-------------------------------------------------------------------------------------------------------+
+| **-K**           | Path to store the manager certificate key.                                                            |
```

Comment:
Let's avoid talking about `manager certificate`. This tool can be used to generate an X.509 certificate for any purpose, even the default one is to create the certs for the manager.